### ISSUE TITLE
add annotated routing to controller

### ISSUE THREAD

**Issue by hxtree** [ouxsoft/Hoopless#23]
https://symfony.com/doc/current/controller.html

```

use Symfony\Component\HttpFoundation\Response;
use Symfony\Component\Routing\Annotation\Route;

class LuckyController
{
    /**
     * @Route("/lucky/number/{max}", name="app_lucky_number")
     */
    public function number(int $max): Response
    {
        $number = random_int(0, $max);

```

**Comment by hxtree** [comment]
maybe first implement for dynamic images

**Comment by hxtree** [comment]
Resolved 179e13fb33208e1af024c4e88d60138a96a65e3b
